corehq/util/view_utils.py

```diff
@@ -68,6 +69,7 @@ def inner(request, *args, **kwargs):
         except BadRequest as e:
             return _get_json_exception_response(400, request, e)
         except Exception as e:
+            notify_exception(request, 'JSON exception response: {}'.format(e))
```

Comment:
The problem with this is it's passing a byte string as the second argument to `notify_exception` without telling it the encoding, so it's not easy for `notify_exception` to do the right thing with it. However, looking at [the source](https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/logging.py#L39-L40), it appears that function isn't handling unicode correctly either... So it's hard to say what the right thing is here.

Maybe [this discussion on exceptions that possibly contain non-ASCII bytes or unicode](https://github.com/dimagi/commcare-hq/pull/4237/files/73076d7ebc1795b05f3d92330a516978b8569a0d#r17075574) might be helpful? Unfortunately a correct solution is kinda kludgy. The benefit of passing a unicode string instead of bytes is that the recipient can do something intelligent with it without guessing at the encoding.